pkg/features/kube_features.go

```diff
@@ -130,7 +130,9 @@ const (
 	MemoryManager featuregate.Feature = "MemoryManager"
 
 	// owner: @sjenning
+	// alpha: v1.4
```

Comment:
sysctl feature is implemented in 1.4
but there is no feature gate, it use an alpha annotation.

IMO, we need not add this line.

Reply:
It's useful for tracking the implementation history. I don't think this comment does any harm.

Reply:
agree. no harm with it.